code/kg2/prompt_for_password_and_save_to_temp_file.py

```diff
@@ -16,10 +16,17 @@
 import getpass
 import os
 import tempfile
-
+os.sys.path.append("..")  # make modules in the code/ directory accessible
```

Comment:
I'd also suggest changing this line to:
```
sys.path.append(os.path.dirname(os.path.abspath(__file__))+"/../")
```
to make it more robust. (That way this script can be run from outside the `/code` directory - isn't dependent on the cwd.)

Reply:
smart!